internal/services/aws_reservation_service.go

```diff
@@ -99,14 +99,28 @@ func CreateAWSReservation(w http.ResponseWriter, r *http.Request) {
 		},
 	}
 
+	// Get Image builder client
+	IBClient, err := clients.GetImageBuilderClient(r.Context())
```

Comment:
This will likely panic as the GetImageBuilderClient is nil, this is missing an import.